rosinstall_noetic_core20/snap/snapcraft.yaml

```diff
@@ -0,0 +1,41 @@
+name: snapped-ros1-pkg # you probably want to 'snapcraft register <name>'
+base: core20 # the base snap is the execution environment for this snap
+version: '0.1' # just for humans, typically '1.2+git' or '1.3.2'
+summary: Basic ROS1 package snap # 79 char long summary
+description: |
+  Basic ROS1 package snap with config and launchfile based on core20
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+
+parts:
+  snapped-ros1-pkg:
+    # See 'snapcraft plugins'
+    plugin: catkin
+    source: .
+    build-packages:
+      - libboost-filesystem1.71.0
+      - liblog4cxx-dev
+      - libboost-regex1.71.0
+      - git
+      - python3-vcstool
+    stage-packages:
+      - ros-noetic-rosbash # necessary if we need rosrun
+      - ros-noetic-roslaunch # necessary if we need roslaunch
+    override-pull: |
+          # retrieve the vcs collection from github
+          #if [ ! -f snapped_ros1_pkg.rosinstall ]; then wget https://raw.githubusercontent.com/Guillaumebeuzeboc/snapped_ros1_rosinstall/main/snapped_ros1_pkg.rosinstall; fi
```

Comment:
Is this meant to be an example of retrieving a rosinstall file from internet? If so, make it clear in the comment. Note also that `vcstool` can handle that directly, e.g.

```bash
vcs import --input https://raw.githubusercontent.com/ros2/ros2/rolling/ros2.repos .
```

Reply:
It is fine to leave a comment showing that the file can be retrieve from internet. Just make it explicit, e.g. `# This demonstrates that the rosinstall file can be retrieve from a URL`.